src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -32,6 +32,8 @@
 import games.strategy.sound.SoundPath;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.ai.proAI.ProData;
+import games.strategy.triplea.ai.proAI.util.ProBattleUtils;
```

Comment:
I missed it, I figured you had finished your review. Iterating on a review is not a good thing, we should both be more clear when we have not finished looking at everything.